server/config/options.go

```diff
@@ -159,6 +160,12 @@ var DefaultConfig = Config{
 			ResponseTime: true,
 		},
 	},
+	Quota: QuotaConfig{
+		Enabled:         true,
```

Comment:
Make sure to bump up the limit for the dev cluster after landing it otherwise it may impact the benchmarking.